digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/ComplementaryLinkDAO.scala

```diff
@@ -15,66 +15,68 @@ import slick.jdbc.{GetResult, PositionedResult}
 
 class ComplementaryLinkDAO extends RoadLinkDAO {
 
+  protected override def withFinNameFilter(roadNameSource: String)(roadNames: Set[String]): String = {
```

Comment:
Toi "Fin" tarkoittaa varmaan tässä suomen kieltä? Jos näin, niin toi nimeäminen on vähän hämäävä, kun tuolla käsitellään noita ruotsinkielisiäkin.

Reply:
Joo tämä on omasta mielestäkin vähän erikoinen nimi. Nimi on peräisin RoadLinkDAO:sta, jossa toiminnallisuus on ihan sama. Tässä pitää override tuo saman niminen funktio RoadLinkDAO:sta, joten siksi valittu kys. nimi.